getting-started/go.md

```diff
@@ -1 +1,112 @@
+
+```go
+package main
+
+import (
+	"github.com/ipfs/go-cid"
+	"github.com/ipld/go-ipld-prime"
+	_ "github.com/ipld/go-ipld-prime/codec/dagcbor"
+	"github.com/ipld/go-ipld-prime/fluent"
+	cidlink "github.com/ipld/go-ipld-prime/linking/cid"
+	basicnode "github.com/ipld/go-ipld-prime/node/basic"
+)
+
+var linkBuilder = cidlink.LinkBuilder{cid.Prefix{
+	Version:  1,    // Usually '1'.
+	Codec:    0x71, // dag-cbor as per multicodec
+	MhType:   0x15, // sha3-384 as per multicodec
+	MhLength: 48,   // sha3-224 hash has a 48-byte sum.
+}}
+```
+
+With that out of the way, we can build our first nodes. Beware that, for the
+sake of brevity, we aren't handling any errors.
+
+```go
+func main() {
+	ctx := context.Background()
+	// For now, we're not storing the blocks anywhere.
+	store := func(ipld.LinkContext) (io.Writer, ipld.StoreCommitter, error) {
+		return ioutil.Discard, func(lnk ipld.Link) error { return nil }, nil
+	}
+
+	eric := fluent.MustBuildMap(basicnode.Prototype.Any, 1, func(na fluent.MapAssembler) {
+		na.AssembleEntry("name").AssignString("Eric Myhre")
+	})
+	ericLink, _ := linkBuilder.Build(ctx, ipld.LinkContext{}, eric, store)
+	people := fluent.MustBuildList(basicnode.Prototype.Any, 1, func(na fluent.ListAssembler) {
+		na.AssembleValue().AssignLink(ericLink)
+	})
+	peopleLink, _ := linkBuilder.Build(ctx, ipld.LinkContext{}, people, store)
+	fmt.Println(peopleLink)
+}
+```
+
+We encode both nodes into blocks with `dag-cbor`, which can encode all JSON
+types, binary data, and IPLD links (CIDs). Finally, the `people` node contains a
+link to Eric's block.
+
+You can also implement a storage layer to save and retrieve blocks. For example,
+here's an in-memory map store:
+
+```go
+func main() {
+	ctx := context.Background()
+	storage := make(map[ipld.Link][]byte)
+	store := func(ipld.LinkContext) (io.Writer, ipld.StoreCommitter, error) {
+		buf := bytes.Buffer{}
+		return &buf, func(lnk ipld.Link) error {
+			storage[lnk] = buf.Bytes()
+			return nil
+		}, nil
+	}
+	loader := func(lnk ipld.Link, _ ipld.LinkContext) (io.Reader, error) {
+		return bytes.NewReader(storage[lnk]), nil
+	}
+
+	eric := fluent.MustBuildMap(basicnode.Prototype.Any, 1, func(na fluent.MapAssembler) {
+		na.AssembleEntry("name").AssignString("Eric Myhre")
+	})
+	ericLink, _ := linkBuilder.Build(ctx, ipld.LinkContext{}, eric, store)
+	daniel := fluent.MustBuildMap(basicnode.Prototype.Any, 1, func(na fluent.MapAssembler) {
+		na.AssembleEntry("name").AssignString("Daniel Martí")
+	})
+	danielLink, _ := linkBuilder.Build(ctx, ipld.LinkContext{}, daniel, store)
+	people := fluent.MustBuildList(basicnode.Prototype.Any, 2, func(na fluent.ListAssembler) {
+		na.AssembleValue().AssignLink(ericLink)
+		na.AssembleValue().AssignLink(danielLink)
+	})
+	peopleLink, _ := linkBuilder.Build(ctx, ipld.LinkContext{}, people, store)
+
+	nb := basicnode.Prototype.Any.NewBuilder()
+	_ = peopleLink.Load(ctx, ipld.LinkContext{}, nb, loader)
+	people2 := nb.Build()
```

Comment:
an idea: it would be neat if there was a single-liner to do this. perhaps something like:

```
people2 := fluent.Load(ctx, ipld.LinkContext{}, peopleLink, basicnode.Prototype.Any, loader)
```

Still a very long line, but at least a single line and not three (requiring a scoped Builder var).